package.json

```diff
@@ -149,10 +152,11 @@
     "package-hash": "^1.1.0",
     "pkg-conf": "^2.0.0",
     "plur": "^2.0.0",
-    "power-assert-context-formatter": "^1.0.4",
-    "power-assert-renderer-assertion": "^1.0.1",
-    "power-assert-renderer-succinct": "^1.0.1",
+    "pretty-format": "github:vadimdemedes/pretty-format#colors",
     "pretty-ms": "^2.0.0",
+    "react": "^15.4.0",
```

Comment:
React is needed by `react-element-to-jsx-string` module. I guess I could try submitting a PR (if possible) to avoid the dependency on React.